jena-ontapi/src/main/java/org/apache/jena/ontapi/OntModelControls.java

```diff
@@ -247,19 +242,14 @@ public enum OntModelControls {
      */
     USE_OWL_PROPERTY_IRREFLEXIVE_FEATURE,
     /**
-     * Controls {@link OWL2#inverseOf owl:inverseOf} functionality.
+     * Controls {@link OWL2#inverseOf owl:inverseOf} functionality (InverseObjectProperty axiom).
      * If disabled,
      * {@link OntObjectProperty#inverseProperties() OntObjectProperty#inverseProperties()}
-     * will return {@code false}, modification operations, such as
+     * will return empty {@code Stream}, modification operations, such as
      * {@link OntObjectProperty#addInverseProperty(OntObjectProperty) OntObjectProperty#addInverseProperty(OntObjectProperty)},
      * will throw {@link OntJenaException.Unsupported OntJenaException.Unsupported} exception.
      */
-    USE_OWL_PROPERTY_INVERSE_OF_FEATURE,
-    /**
-     * If this key is set to {@code true}, an anonymous inverse object property type is enabled (OWL2 feature).
-     * That is {@code _:x owl:inverseOf PN}, where {@code _:x} is a b-node and {@code PN} is a named object property.
-     */
-    USE_OWL_INVERSE_OBJECT_PROPERTY_FEATURE,
+    USE_OWL_INVERSE_OBJECT_PROPERTIES_FEATURE,
```

Comment:
Something like this ?
```java
    /**
     * @deprecated this setting doesn't work anymore, use {@link #USE_OWL_INVERSE_OBJECT_PROPERTIES_FEATURE} instead
     */
    @Deprecated
    USE_OWL_PROPERTY_INVERSE_OF_FEATURE,
    /**
     * @deprecated this setting doesn't work anymore, the control delegated directly to OntPersonality
     */
    @Deprecated
    ALLOW_ANONYMOUS_INDIVIDUALS,
    /**
     * @deprecated this setting doesn't work anymore, the control delegated directly to OntPersonality
     */
    @Deprecated
    USE_OWL_INVERSE_OBJECT_PROPERTY_FEATURE,
```

Reply:
Using a `@Deprecated` marking usually means "it works for now but will not at some point". 

If the constant no longer has any effect, that is, there is a change in behaviour, then it might as be removed.
There is only so much effort worth spending on leaving the old way in place. This module is new and we did categorise it as experimental.

If it is effort and distorts the code, then just remove them.

BTW: `@Deprecated(forRemovel = true)`

Reply:
I doubt that anyone is already using this api, if so, then in a very simple manner - these options are very advanced way to configure model. I think simply removing these constants is the best way now. It is quite difficult to bring back their original behavior without code degradation.